apps/web/src/app/(signing)/sign/[token]/signature-field.tsx

```diff
@@ -176,6 +192,41 @@ export const SignatureField = ({
     }
   };
 
+  useEffect(() => {
```

Comment:
Had a bit of difficulty trying to fit the text inside the Signature field. 😬

Reply:
You might want a layout effect here to block rendering until you get a real number, see: https://sentry.engineering/blog/perfectly-fitting-text-to-container-in-react